components/script/dom/document.rs

```diff
@@ -3627,6 +3627,10 @@ impl Document {
             task.run_box();
         }
     }
+    /// Returns whether there are any active script or layout blockers.
```

Comment:
I'm not sure this is actually necessary, but you should separate the function with a newline here.
```suggestion

    /// Returns whether there are any active script or layout blockers.
```

Reply:
I agree this function is quite straight forward...